torch/csrc/jit/codegen/cuda/manager.cpp

```diff
@@ -177,48 +173,10 @@ void runCudaFusionGroup(const Node* fusion_node, Stack& stack) {
       }
       ShapeTypePropagate(graph);
     }
-    /*
-    // TODO: Delete the shape inference here once we switch to
-    //       ExpressionEvaluator to allocate outputs
-    // shape inference in graph to allocate outputs
-    // update shape information per the new inputs;
-    EraseShapeInformation(shape_inf_graph);
-    for (size_t i = 0; i < nInputs; i++) {
-      shape_inf_graph->inputs()[i]->setType(inputs[i].type());
-    }
-    // shape inference
-    ShapeTypePropagate(shape_inf_graph);
-
-    // we need to construct outputs;
-    std::vector<at::Tensor> outputs;
-    for (const auto* output : shape_inf_graph->outputs()) {
-      const auto type = output->type()->expect<TensorType>();
-      // Expect output to be tensor;
-      TORCH_CHECK(
-          type && type->isComplete(),
-          "Complete TensorType for output is expected.");
-
-      const auto device = *(type->device());
-      const auto scalar_type = *(type->scalarType());
-
-      auto options = at::TensorOptions()
-                         .dtype(scalar_type)
-                         .layout(at::kStrided)
-                         .device(device)
-                         .requires_grad(type->requires_grad());
-
-      // TODO: We should infer output shape from `inputs`
-      const auto sizes = extractSizes(type);
-      const auto strides = extractStrides(type);
-
-      const auto tensor = at::empty_strided(sizes, strides, options);
-      outputs.push_back(tensor);
-    }
-    CudaFusionManager::getManager().runFusionNode(
-        kernel_id, graph, inputs, outputs);
-    */
+
     FusionExecutor executor;
     auto fusion = parseJitIR(graph);
+    fusion->printMath();
```

Comment:
NOTE TO MYSELF: need to switch to `runFusionNode` so we actually go through our naive kernel cache, instead of codegen a new kernel every iteration.